src/helix-ui/styles/core/hx-alert-action.less

```diff
@@ -0,0 +1,15 @@
+hx-alert-action {
```

Comment:
Actions should be triggered via buttons rather than links.
Can we use a CSS class instead of an element? _(This would move styles to `modules/alert.less`)_